content/200-concepts/100-components/01-prisma-schema/index.mdx

```diff
@@ -159,7 +159,7 @@ DATABASE_URL=postgresql://test:test@localhost:5432/test
 DATABASE_URL_WITH_SCHEMA=${DATABASE_URL}?schema=public
 ```
 
-There are a few limitations with `env` at the moment:
+There are a few limitations with `env()` at the moment:
 
 - It is not possible to use environment variables for the `provider` argument in `datasource` and `generator` definitions.
```

Comment:
This will not go away, and it technically not a limitation: The `env()` function is only meant to be used in a few specific properties. We should have those documented somewhere as well.

Reply:
I think something has been lost during a merge here as I had already changed this part

Reply:
Yep! just as i thought, i override my changes. Have added back:

The `env()` function can be used in the following places:

- A datasource url
- Generator binary targets